devicetypes/smartthings/zigbee-motion-sensor-light.src/led-cpx-light.groovy

```diff
@@ -0,0 +1,167 @@
+/**
+ *  Copyright 2022 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  LED CPX light
+ *
+ *  Author: SAMSUNG LED
+ *  Date: 2022-01-05
+ */
+
+metadata {
+	definition(name: "LED CPX light", namespace: "SAMSUNG LED", author: "SAMSUNG LED", runLocally: true) {
```

Comment:
I don't think events will get properly dispatched to the child sensor if you have `runLocally` set to `true`

Reply:
If I delete "runLocally: true", rather plug-in works strangely. It doesn't match to on/off status of device, plus fail to initializing.